Source/WebInspectorUI/UserInterface/Models/Resource.js

```diff
@@ -1250,17 +1244,17 @@ WI.Resource = class Resource extends WI.SourceCode
         let lines = [];
 
         let protocol = this.protocol || "";
-        if (protocol === "h2") {
+        if (!protocol || protocol === "http/1.1") {
```

Comment:
id suggest flipping this logic around and having the fallback path be the oldest "version"
```js
if (protocol === "h2" || protocol === "h3") {
    // HTTP/2 and HTTP/3 Request pseudo headers:
    ...
} else {
    // HTTP/1.1 request line
    ...
}
```

Reply:
The change is deliberate. There might be a minor version bump HTTP/2.1 soon, and eventually there will be HTTP/4. I'd prefer that they take the HTTP/2 path than the legacy HTTP/1 path.

Reply:
are we certain that HTTP/4 will use the same pseudo headers?

i have no problem with something like `protocol.startsWith("h2")` in order to capture `h2.1` (assuming that even is the right string)

Reply:
Well, I can't tell what exactly h4 will be like. My best guess is that pseudo header fields will stay since they've been working fine, and there hasn't been any proposals to change it that I'm aware of. The next major version of HTTP might involve binary structured fields, changes to header/trailer field compression, and possibly QUICv2. One thing for sure is that it can't revert back to h1-style requests / responses.

`h2.1` is one of the current proposals. Other people are still exploring using SETTINGS frame or other ways to change the protocol behavior but retain the protocol version.